dashboard/DashboardDB.py

```diff
@@ -227,6 +312,21 @@ def _create_queue_table(self, cursor, hpc: const.HPC):
             )
         )
 
+    def _create_quota_table(self, cursor, hpc: const.HPC):
```

Comment:
No need to make this a function as it is only ever called from a single location